.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Fair enough. I always considered it unnecessary boilerplate but with a bigger contributor base telling everyone to use a global .gitignore is pretty tedious. But then we should have the same .gitignore in all projects and I'd also argue to include IDE settings (I keep publishing my `.idea` folder to NPM because it doesn't honour the global .gitignore or have to add an unnecessary separate .npmignore).